homework3/task4.js

```diff
@@ -0,0 +1,15 @@
+function getFactorial(num) {
+  let fact = 1;
+  if (typeof num === "number") {
+    for (n = 2; n <= num; n++) {
```

Comment:
try to add `let` before `n`. Otherwise that variable will be available outside `for loop`

Reply:
Ohh thanks .. I didn’t noticed it…